src/app/state/transaction/types.ts

```diff
@@ -56,6 +56,15 @@ export interface TransactionState {
   success: boolean
   error?: ErrorPayload
   preview?: TransactionPreview
+  // TODO: Should pendingTransactions be persisted?
```

Comment:
I wouldn't, and I wouldn't even keep them if user switches accounts

I'd just show "has pending transactions" warning after reload by checking account nonce and nonces in latest transactions (but would need to fetch tx details for each transaction in the list when we get them from oasisscan)